input/tps-egfr-prizes.txt

```diff
@@ -1,4 +1,4 @@
-NODEID	prize	sources	targets	active
+NODEID	prize	sources	targets	active	dummy
```

Comment:
Can you also please modify the EGFR config file to use this dummy node column so EGF is the only selected dummy node?

Reply:
the `egfr.yaml` file has the following oi1 params:
```
   name: omicsintegrator1
    params:
      include: true
      run1:
        b:
          - 0.55
          - 2
          - 10
        d:
          - 10
        g:
          - 1e-3
        r:
          - 0.01
        w:
          - 0.1
        mu:
          - 0.008
        dummy_mode: ["file"]
```

so i believe that with the dummy column and `EGF_HUMAN` `dummy` column set to `True` should do this?